cves/kernel/CVE-2019-13648.yml

```
@@ -241,8 +247,8 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
+  - commit: d2b9d2a5ad5ef04ff978c9923d19730cb05efd55
+    note: Appears to be an early detection to the vulnerability. Unclear why it took so long for the bug to manifest and still cause issues 4 years later
```

Wrapped to 80 char line limit. Small modifications.
```suggestion
    note: |
      Appears to be an early detection of the vulnerability. However, it's
      unclear why it took so long for the bug to manifest and cause issues four
      years later.
```

---

Replaced